src/home/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import {
+  Card, CardImg, CardText, CardBody,
+  CardTitle, CardSubtitle, Button
+} from 'reactstrap';
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+const Product = ({title,price,description,image})=>{
```

Comment:
Putting a `product` here is not correct. So please extract this method into a `Product` component and use it inside our `Home` component.